National_Precinct_Bound_10_07_2022/README.md

```diff
@@ -0,0 +1,20 @@
+# 2020 National Precinct Boundary Shapefile 10_07_2022
+
+## Background:
+- We received a data request asking for a 2020 national precinct shapefile.
+
+## Approach:
+- For every state load in precinct boundary and election result shapefile originally from VEST hosted on the RDH website.
```

Comment:
I would add a comment about the states (KY and NJ?) where there were multiple files and just explain which file you chose and why

Reply:
Yeah I will add that in.